index.html

```diff
@@ -55,5 +55,175 @@ <h1 class="headline__title">I’m Ahmed Glad to see you!</h1>
         </li>
       </ul>
     </section>
+
+    <section class="works">
+      <div class="works__single">
+        <img
+          class="works__single__img"
+          src="images/project2.png"
+          alt="project 1"
+        />
+
+        <p class="works__single__title">Multi-Post Stories</p>
+
+        <div class="works__single__details">
+          <p
+            class="works__single__details__single works__single__details__single--dark"
+          >
+            CANPOY
+          </p>
+          <div class="dot"></div>
+          <p
+            class="works__single__details__single works__single__details__single--light"
+          >
+            Back End Dev
+          </p>
+          <div class="dot"></div>
+          <p
+            class="works__single__details__single works__single__details__single--light"
+          >
+            2015
+          </p>
+        </div>
+
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+
+        <div class="works__single__tags">
+          <div class="works__single__tags__single">html</div>
+          <div class="works__single__tags__single">css</div>
+          <div class="works__single__tags__single">javascript</div>
+        </div>
+
+        <button class="button">See Project</button>
+      </div>
+
+      <div class="works__single">
+        <img
+          class="works__single__img"
+          src="images/project1.png"
+          alt="project 1"
+        />
+
+        <p class="works__single__title">Tonic</p>
+
+        <div class="works__single__details">
+          <p
+            class="works__single__details__single works__single__details__single--dark"
+          >
+            CANPOY
+          </p>
+          <div class="dot"></div>
+          <p
+            class="works__single__details__single works__single__details__single--light"
+          >
+            Back End Dev
+          </p>
+          <div class="dot"></div>
+          <p
+            class="works__single__details__single works__single__details__single--light"
+          >
+            2015
+          </p>
+        </div>
+
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+
+        <div class="works__single__tags">
+          <div class="works__single__tags__single">html</div>
+          <div class="works__single__tags__single">css</div>
+          <div class="works__single__tags__single">javascript</div>
+        </div>
+
+        <button class="button">See Project</button>
+      </div>
+    </section>
+
+    <section class="about">
+      <h1 class="headline__title">About Myself</h1>
```

Comment:
- [ ] [REQUIRED] Please change this `<h1>` tag to any of the tags between `<h2>` and `<h6>` because you can only use one h1 tag per page for proper SEO.
- [ ] [REQUIRED] Please also ensure you stick to the original design as much as possible. Fix the 'About Myself Section' to match the original.

### Your Design
![three](https://user-images.githubusercontent.com/63964646/148439905-cae003ff-5ba6-439e-86ef-6acd9844ba97.PNG)

### Original Design
![four](https://user-images.githubusercontent.com/63964646/148439914-afc1320f-fc85-4db3-ad95-fb4854a18813.PNG)